enterprise/internal/database/perms_store_test.go

```diff
@@ -280,11 +279,13 @@ func checkRegularPermsTable(s *permsStore, sql string, expects map[int32][]uint3
 		if expects[id] == nil {
 			return errors.Errorf("unexpected row in table: (id: %v) -> (ids: %v)", id, intIDs)
 		}
-		want := fmt.Sprintf("%v", expects[id])
 
-		have := fmt.Sprintf("%v", intIDs)
-		if have != want {
-			return errors.Errorf("intIDs - key %v: want %q but got %q", id, want, have)
+		comparator := func(a, b uint32) bool {
+			return a < b
+		}
+
+		if cmp.Diff(expects[id], intIDs, cmpopts.SortSlices(comparator)) != "" {
+			return errors.Errorf("intIDs - key %v: want %q but got %q", id, expects[id], intIDs)
```

Comment:
IIRC `%q` only works for strings, which were, but we have `[]uint32` now.
```suggestion
			return errors.Errorf("intIDs - key %d: want %d but got %v", id, expects[id], intIDs)
```